pages/user/profile/personal.tsx

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+
+import { Stack, Typography } from "@mui/material";
+
+import EditorTabLayout from "../../../components/Profile/EditorTabLayout";
+import PersonalInfoForm from "../../../components/Profile/PersonalInfoForm";
+import { usePrivateProfile } from "../../../hook/profile/usePrivateProfile";
+import { useAuthState } from "../../../hook/useAuthState";
+import { UserPrivateProfile } from "../../../interfaces/user";
+
+const PersonalProfilePage = () => {
+  const router = useRouter();
+  const { authState } = useAuthState();
+  const [privateProfile, updateProfile] = usePrivateProfile(false);
```

Comment:
<img width="1919" height="923" alt="Image" src="https://github.com/user-attachments/assets/815d58f0-69cb-4c19-898c-e20f2a10bdc9" />

初回登録時にもかかわらずPrivateProfileの取得エラーが発生しました。
フックの実装を見た感じ、取得エラーに関しては初回登録時のみフックの `isRegister` フラグを `true` にすれば取得を省略してエラーが起こらなくなると思います！

[参考: hook/profile/usePrivateProfile.ts](https://github.com/SIT-DigiCre/digicore_v3_frontend/blob/f44f45d8a18df2449d0bf7cf99799be670fdc647/hook/profile/usePrivateProfile.ts#L12-L18)

```suggestion
  const [privateProfile, updateProfile] = usePrivateProfile(true);
```